lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -62,6 +60,7 @@ export default function BasicLtiLaunchApp() {
       // to third party APIs (eg. the LMS's file storage).
       via_url_callback: viaUrlCallback,
     },
+    canvas: { authUrl, speedGrader },
```

Comment:
If we are going to pop these out into variables, they should probably be `canvasAuthUrl` and `canvasSpeedGrader` otherwise we are back to using generic terms for Canvas specific things.

Less so for Speed Grader, but it still requires some domain specific knowledge, so why not just always map the names directly from the path?

Reply:
Done. Note that generally, though, the frontend is absolutely rife with generic names for specific things and it's quite a pain to work with, refactoring it would take too long and the risk of breaking things would be too high, so I'm going to be keeping frontend changes to a minimum